### PR TITLE
Remove unused files & add MacOS instructions

### PR DESCRIPTION
**Pull request by cpeel**
This MR does two notable things:
1. Removes unused perl modules
2. Adds instructions for how to make Guiguts work on MacOS

### CODE REVIEW

**Comment by cpeel** [comment]
@srjfoo - I would appreciate validation of the MacOS instructions. These instructions seemed to work for me.

**Comment by srjfoo** [comment]
I'll be happy to!

**Comment by cpeel** [comment]
Unless you also installed Tk::Toolbar from CPAN into your perl installation, this is expected. Guiguts still needs that and Tk::LineNumberText -- we just want it to get and use the CPAN version instead of bundling it in.

**Comment by windymilla** [comment]
Yep. That makes sense

**Comment by cpeel** [comment]
I removed the commit that removed the CPAN modules. I want to push that in with another commit that address some of the larger Windows/perl question.

**Comment by windymilla** [comment]
Not important, but maybe of interest: my GG is running happily without lib/Tk/LineNumberText, though it does have lib/Guiguts/LineNumberText (looks like an edited/stripped down version)

**Comment by cpeel** [comment]
> Not important, but maybe of interest: my GG is running happily without lib/Tk/LineNumberText, though it does have lib/Guiguts/LineNumberText (looks like an edited/stripped down version)

Indeed - it works for me without that too. I've removed it from the list of needed modules.

This branch is now rebased off of HEAD.

**Comment by cpeel** [comment]
After more testing (and with help from @srjfoo) I've flushed out the [MacOS instructions](https://github.com/cpeel/guiguts/blob/remove-unused-files/INSTALL.md#macos) more. I've also added a helper script to install the perl modules which we should be able to reuse on Windows as well.

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+perl guiguts.pl
+```
+
+## Perl Modules
+
+Guiguts requires the following Perl modules to be installed via CPAN:
+
+* LWP::UserAgent
+* Tk
+* Tk::ToolBar
```
Comment (srjfoo) [inline INSTALL.md:140]: `HTML::TokeParser` isn't actually needed, after all? (That seems to have disappeared form the instructions.)

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Installation
+
+Upgrading requires a complete reinstall (save `header.txt`, `setting.rc`,
+and any files you wish to keep in the `data/` directory, although in
+theory these should not be overwritten).
+
+Please direct any help requests to the
+[DP forum Guiguts thread](http://www.pgdp.net/phpBB2/viewtopic.php?t=3075).
```
[the remark this answers is not in the document]
Comment (cpeel) [reply INSTALL.md:8]: Yes, I'll get that updated.

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+using `cpanm`.
+
+For example:
+```
+cpanm --notest --install Tk
+```
+
+Now unzip `guiguts-n.n.n.zip` and start Guiguts with:
+```
+perl guiguts.pl
```
[the remark this answers is not in the document]
Comment (cpeel) [reply INSTALL.md:123]: Also a good idea!

File: INSTALL.md (hunk 1 above)
Comment (cpeel) [reply INSTALL.md:140]: It is needed but as several people have pointed out, it is included with the base perl install for both Windows and Mac so I opted not to call it out (like we don't call out all the other modules needed but included with perl).

Comment (srjfoo) [reply INSTALL.md:141]: Makes sense -- I hadn't realized it was part of the base perl install, just noticed the difference between these instructions and the earlier ones.

**Comment by cpeel** [comment]
We've had some good reports on the Mac instructions, so I think this is good for merging pending OK by @windymilla.

**Comment by windymilla** [comment]
Yes, happy with this

**Pull request merged**